static/js/zamboni/devhub.js

```diff
@@ -885,8 +885,8 @@ function initCatFields(delegate) {
             $("input", $main).attr("checked", false).attr("disabled", false);
         };
         checkMainDefault();
-        $('input', $main).live('change', checkMain);
-        $('input', $misc).live('change', checkOther);
+        $('input', $main).on('change', checkMain);
```

Comment:
I could be wrong, but having the 2nd arg is the same as concatenating the two, right? So this could be written `$(document).on('change', $main + ' input', checkMain)` instead? Not sure about that, but if that's the case, maybe it would be better that way?